dev/com.ibm.ws.kernel.feature.resolver_fat/fat/src/com/ibm/ws/feature/tests/FeatureDetailsUnitTest.java

```diff
@@ -379,4 +428,46 @@ public boolean validate(String symName,
         validate("Validate feature is-compatibility", isCompatibilityTester);
     }
 
+    /**
+     * Verify that 'instantOnEnabled' is set to true for all versionless features.
+     */
+    @Test
+    public void features_validateInstantOnTest() throws Exception {
```

Comment:
When this fails all you will see is:

```
features_validateInstantOnTest:junit.framework.AssertionFailedError: Validate feature instantOnEnabled ... failed
	at com.ibm.ws.feature.tests.FeatureDetailsUnitTest.validate(FeatureDetailsUnitTest.java:243)
	at com.ibm.ws.feature.tests.FeatureDetailsUnitTest.features_validateInstantOnTest(FeatureDetailsUnitTest.java:471)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
```

Not indicating what failed.  You must then look at the output.txt of the bucket to see some prints of the feature names.  It would be nice if com.ibm.ws.feature.tests.FeatureDetailsUnitTest.validate(String, FeatureTester) didn't just return boolean, but returned other data that can be gathered into the final failure message indicating what feature(s) caused the problem.